FXSkins/src/main/resources/com/pixelduke/control/skin/fx-skins.css

```diff
@@ -47,23 +47,31 @@ TextField {
     -fx-skin: "com.pixelduke.control.skin.FXTextFieldSkin";
 }
 
-TextField {
-    -right-button-visible: true;
+.text-field {
+    -fx-padding: 0.3333333em 3em 0.3333333em 0.5833333em;
 }
 
-TextField > .right-button > .right-button-graphic {
+.text-field > .right-button > .right-button-graphic {
     -fx-shape: "M221.738,305.873l6.135,6.16l-2.875,2.863l-6.135-6.159l-6.263,6.237l-2.864-2.875l6.263-6.238l-6.177-6.202l2.875-2.863l6.177,6.201l6.244-6.22l2.864,2.876L221.738,305.873z";
 
     -fx-padding: 0.416667em;
-    -fx-background-color: #333;
+    -fx-background-color: #111;
 }
 
-TextField > .right-button:hover {
+.text-field > .right-button {
     -fx-cursor: default;
+
+    -fx-background-color: transparent;
+    -fx-background-insets: -0.3333333em -0.6666666em -0.3333333em -0.5833333em; /* 4 8 4 7 px in em */
+    -fx-background-radius: 0 3 3 0, 2, 2;
 }
 
-TextField > .right-button:hover > .right-button-graphic {
-    -fx-background-color: #666666;
```

Comment:
Why are we removing this?

Reply:
This was removed to make the button more in line with the password textfield. The button only changes when you press (and hold) the button.

Reply:
I mean we could add hover back. But in my opinion they should either both have a hover effect or none. And my recurring example of the windows settings search bar also does not have a hover effect

Reply:
That's a good point. In that case I would prefer to also add the hover effect on the passwordfield. We don't need to make an exact copy of Microsoft fluent design, you'll see that JMetro doesn't copy it in several places (where I don't like the way fluent design looks or its UX). One example: Fluent Design Spinners don't look so good in my opinion and JMetro has a completely different look for them.
JMetro is inspired by FD but doesn't pretend to be a copy of it. 
Also FXSkins is a general purpose library for adding better skins to existing apps. JMetro will use it but so will other apps not using JMetro. My idea is that by default FXSkins looks good with Modena (the default javafx theme).

Reply:
I get that. But do you agree to add them back on both?

Reply:
So, as you suggested, and I think that's a good idea: let's also add the hover effect to the password field "right button". Sound good?